languages/js/src/Query.ts

```diff
@@ -135,22 +139,27 @@ export class Query {
     // have a nice hook where we know every class has been registered
     // (e.g., once we enforce that all registerCalls() have to happen
     // before loadFiles()).
-    const typ = this.#host.getType(rel.otherType)!;
+    const typ = this.#host.getType(rel.otherType);
+    if (typ === undefined) throw new UnregisteredClassError(rel.otherType);
+
+    if (!isObj(receiver))
```

Comment:
Think I'm using `isObj()` a bit too liberally, potentially leading to false negatives. Think I really want to check if the value is _indexable_; I don't particularly care that it responds to `typeof` with `'object'`.

Reply:
c83d92a8